ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/KeyValueConsumption.scala

```diff
@@ -37,28 +37,28 @@ object KeyValueConsumption {
     * @param entry: The log entry.
     * @return [[Update]] constructed from log entry.
     */
-  def logEntryToUpdate(entryId: DamlLogEntryId, entry: DamlLogEntry): Option[Update] = {
+  def logEntryToUpdate(entryId: DamlLogEntryId, entry: DamlLogEntry): List[Update] = {
 
     val recordTime = parseTimestamp(entry.getRecordTime)
 
     entry.getPayloadCase match {
       case DamlLogEntry.PayloadCase.PACKAGE_UPLOAD_ENTRY =>
-        Some(
-          Update.PublicPackagesUploaded(
-            entry.getPackageUploadEntry.getArchivesList.asScala.toList,
+        entry.getPackageUploadEntry.getArchivesList.asScala.toList.map { archive =>
```

Comment:
Minor: instead of doing `toList` and then `map` you can do both things at once by passing `scala.collection.breakOut` as the implicit parameter. More info [here](https://docs.scala-lang.org/tutorials/FAQ/breakout.html).

Reply:
Done